starfish/pipeline/registration/fourier_shift.py

```diff
@@ -25,7 +25,7 @@ def add_arguments(cls, group_parser):
     def register(self, stack):
         import numpy as np
 
-        mp = stack.max_proj(Keys.CH)
+        mp = stack.max_proj(Keys.CH, Keys.ZLAYER)
```

Comment:
hmmm. add a #TODO here "we should think harder how fourier registration should work in 3D"